Bottom: 9df2d0dc04a1eaa54b56f9df5d1f1b85aec2daf9
Top:    3a12ac36030dc31aae958ea66a0148fdd38e40cd
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-01-24 19:09:53 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
